lib/cn/cn_tree.c

```diff
@@ -103,8 +101,8 @@ cn_node_alloc(struct cn_tree *tree, uint64_t nodeid)
     INIT_LIST_HEAD(&tn->tn_link);
     INIT_LIST_HEAD(&tn->tn_kvset_list);
 
-    atomic_init(&tn->tn_compacting, 0);
-    atomic_init(&tn->tn_busycnt, 0);
+    atomic_set(&tn->tn_compacting, 0);
```

Comment:
Why atomic_set over atomic_init?

Reply:
It turns out atomic_init() is from C11 atomics, HSE only has an atomic_set().  I haven't yet figured out if atomic_init() is necessary, I know they deprecated all their other "INIT" macros...

Reply:
Didn't we already move to C11 atomics? I thought our hse_util/atomic.h is just a wrapper over C11 atomics.

Reply:
Yeah, we did.  But we didn't adopt atomic_init().  Maybe we should?  Dunno, I need to read up on it..
But as Alex noted it's better to clearly mark our atomics, and I agree, which I've done here.  The sneaky
thing is if you don't use our atomic wrappers then the atomics will just work as called out by C11, which
is to use CST semantics by default.  Fortunately, that's much stronger than our general use of relaxed atomics, so we should be good on that front.

Reply:
I think whether atomic_init() is needed or not is implementation dependent, not sure..

Reply:
Maybe for dynamic initialization?  Here they make a case for it being unnecessary:
https://developers.redhat.com/blog/2016/01/19/toward-a-better-use-of-c11-atomics-part-2